pkg/storage/bloom/v1/merge.go

```diff
@@ -76,15 +76,15 @@ func (mbq *HeapIterator[T]) pop() (T, bool) {
 			return mbq.zero, false
 		}
 
-		cur := mbq.itrs[0]
-		if ok := cur.Next(); !ok {
+		curr := mbq.itrs[0]
+		if ok := curr.Next(); !ok {
```

Comment:
don't like my naming? 😭

Reply:
😂

Reply:
This happened accidentally when I reverted a temporary change.